src/components/Catalog/catalog.jsx

```diff
@@ -0,0 +1,87 @@
+import { useState } from "react";
+import database from "./Images/database-icon.png";
+import smartphone from "./Images/smartphone-icon.png";
+import code from "./Images/code-icon.png";
+import layout from "./Images/layout-icon.png";
+import './catalog.css';
+const Catalog = () => {
+  const [catalogs, setCatalog] = useState([
+    {
+      image: smartphone,
+      title: "Web & Mobile Development",
+      body: "Master the art of web and mobile application development. Create dynamic, responsive websites and apps that leave a loating impressions.",
+      id: 1,
+    },
+    {
+      image: code,
+      title: "Programming Essential",
+      body: "Dive into the world of coding with our beginner-friendly programming courses. Learn the fundamental of popular programming languages and build a strong foundation.",
+      id: 2,
+    },
+    {
+      image: layout,
+      title: "UI/UX Design",
+      body: "Design user friendly interfaces and captivating user experiences. Develop the skills to enable desigs that resonate with users.",
+      id: 3,
+    },
+    {
+      image: database,
+      title: "Data Analysis",
+      body: "Make data-driven decisions. Our data analysis courses will teach you how to extract insights frown data and drive business success",
+      id: 4,
+    },
+    {
+      image: database,
+      title: "Data Analysis",
+      body: "Make data-driven decisions. Our data analysis courses will teach you how to extract insights frown data and drive business success",
+      id: 5,
+    },
+    {
+      image: layout,
+      title: "Graphics Design",
+      body: "Design user friendly interfaces and captivating user experiences. Develop the skills to enable desigs that resonate with users.",
+      id: 6,
+    },
+    {
+      image: code,
+      title: "Programming Essential",
+      body: "Dive into the world of coding with our beginner-friendly programming courses. Learn the fundamental of popular programming languages and build a strong foundation.",
+      id: 7,
+    },
+    {
+      image: smartphone,
+      title: "Web & Mobile Development",
+      body: "Master the art of web and mobile application development. Create dynamic, responsive websites and apps that leave a loating impressions.",
+      id: 8,
+    },
+  ]);
```

Comment:
- [ ] There no need to use `useState` here. We don't make any state changes on the page at this moment. The best approach for this is to create a file say `catalogData.js` then create an array of object and import it inside your `Catalog` component to use. I should look something similar to this. :point_down: 

```
  export const catalogData = [
     {
      image: smartphone,
      title: "Web & Mobile Development",
      body: "Master the art of web and mobile application development. Create dynamic, responsive websites and apps that leave a loating impressions.",
      id: 1,
    },
    {
      image: code,
      title: "Programming Essential",
      body: "Dive into the world of coding with our beginner-friendly programming courses. Learn the fundamental of popular programming languages and build a strong foundation.",
      id: 2,
    },
    {
      image: layout,
      title: "UI/UX Design",
      body: "Design user friendly interfaces and captivating user experiences. Develop the skills to enable desigs that resonate with users.",
      id: 3,
    },
    {
      image: database,
      title: "Data Analysis",
      body: "Make data-driven decisions. Our data analysis courses will teach you how to extract insights frown data and drive business success",
      id: 4,
    },
    {
      image: database,
      title: "Data Analysis",
      body: "Make data-driven decisions. Our data analysis courses will teach you how to extract insights frown data and drive business success",
      id: 5,
    },
    {
      image: layout,
      title: "Graphics Design",
      body: "Design user friendly interfaces and captivating user experiences. Develop the skills to enable desigs that resonate with users.",
      id: 6,
    },
    {
      image: code,
      title: "Programming Essential",
      body: "Dive into the world of coding with our beginner-friendly programming courses. Learn the fundamental of popular programming languages and build a strong foundation.",
      id: 7,
    },
    {
      image: smartphone,
      title: "Web & Mobile Development",
      body: "Master the art of web and mobile application development. Create dynamic, responsive websites and apps that leave a loating impressions.",
      id: 8,
    },  
  ]
```

You can create the file inside your catalog folder.